Shopify/hydrogen
# add @total-typescript/ts-reset to skeleton and demo-store

### WHAT is this pull request doing?

Adds [@total-typescript/ts-reset](https://github.com/total-typescript/ts-reset) to demo-store and skeleton templates

### HOW to test your changes?

add this code to any route / component and check the type
```ts
const test = [1, undefined, null, 3, fal, 'test'].filter(Boolean)
```
